src/call/server.rs

```diff
@@ -251,9 +251,10 @@ impl<T> Stream for RequestStream<T> {
     }
 }
 
-// A helper macro used to implement server side unary sink.
-// Not using generic here because we don't need to expose
-// `CallHolder` or `Call` to caller.
+/// A helper macro used to implement server side unary sink.
+/// Not using generic here because we don't need to expose
+/// `CallHolder` or `Call` to caller.
+// TODO: Use type alias to be friendly for documentations.
```

Comment:
documentations -> documentation
Note: Documentation is a set of documents. "documentations" is rare.